rxandroid/src/main/java/io/reactivex/android/plugins/RxAndroidPlugins.java

```diff
@@ -21,39 +23,38 @@
  * Utility class to inject handlers to certain standard RxAndroid operations.
  */
 public final class RxAndroidPlugins {
-    private static volatile Function<Scheduler, Scheduler> onInitMainThreadHandler;
+
+    private static volatile Function<Callable<Scheduler>, Scheduler> onInitMainThreadHandler;
     private static volatile Function<Scheduler, Scheduler> onMainThreadHandler;
 
-    public static void setInitMainThreadSchedulerHandler(Function<Scheduler, Scheduler> handler) {
+    public static void setInitMainThreadSchedulerHandler(Function<Callable<Scheduler>, Scheduler> handler) {
         onInitMainThreadHandler = handler;
     }
 
-    public static Scheduler initMainThreadScheduler(Scheduler scheduler) {
-        Function<Scheduler, Scheduler> f = onInitMainThreadHandler;
-        if (f == null) {
-            return scheduler;
+    public static Scheduler initMainThreadScheduler(Callable<Scheduler> scheduler) {
+        if(scheduler == null) {
```

Comment:
Is there a project style settings file (IntelliJ/AS format) available? I've been manually formatting which can lead to missing stuff like this.